package.json

```diff
@@ -86,7 +86,6 @@
     "node-polyfill-webpack-plugin": "4.0.0",
     "postcss": "^8.4.49",
     "postcss-flexbugs-fixes": "^5.0.2",
-    "postcss-global-import": "^1.0.6",
```

Comment:
please also remove the "overrides" block, as "postcss-global-import" was this dependency was the cause for putting postcss in overrides